### PR TITLE
http://rpc_youthgroup.surge.sh/

### PR DESCRIPTION
**Pull request by rachel-mclean**

### CODE REVIEW

File: public/main.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  height: 3%;
+  margin-top: 20px
+}
+
+.bg-0 {
+  background-color: rgb(108, 152, 167);
+  color: rgb(32, 28, 28);
+}
+
+.row {
```
Comment (jfarmer) [inline public/main.css:29]: Careful about doing this sort of thing. The CSS framework assumes that internal classes like `.row` and `.column` behave a certain way. Overriding their default behavior can have unpredictable consequences.

File: public/main.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  background-color: rgb(108, 152, 167);
+  color: rgb(32, 28, 28);
+  text-align: center;
+}
+
+h1 {
+  color: rgb(32, 28, 28);
+}
+
+h3 {
```
Comment (jfarmer) [inline public/main.css:48]: Let the CSS framework set the margin / padding. It's designed to align everything just right.

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+</head>
+<body>
+  <nav class="navbar navbar-default">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a class="navbar-brand" style="background-color: rgb(253, 179, 108)" href="#">RPC Youth Group</a>
+      </div>
+      <ul class="nav navbar-nav">
```
Comment (jfarmer) [inline public/index.html:20]: Shrink the width of your browser window. There's a range of widths where these nav elements align in a funny way, e.g., https://share.getcloudapp.com/mXuqOx2r

You should be able to use the right combo of bootstrap classes to get the behavior you want.

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Hello, Rachel</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
```
Comment (jfarmer) [inline public/index.html:8]: Ah, careful what version of Bootstrap you're using! The most recent version is 4.4. 3.4.1 is pretty old.

It being old isn't a problem _per se_, but keep it in mind as you're reading documentation. If you're reading something for 4.4, it might not work with 3.4 (e.g., different class names).

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+      <ol class="carousel-indicators">
+        <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+        <li data-target="#myCarousel" data-slide-to="1"></li>
+        <li data-target="#myCarousel" data-slide-to="2"></li>
+      </ol>
+
+      <!-- Wrapper for slides -->
+      <div class="carousel-inner">
+        <div class="item active">
+          <img src="img/door.jpg" alt="" style="width: 25%; height: 25%">
```
Comment (jfarmer) [inline public/index.html:55]: It's kind of jarring that the carousel images change height as you scroll. You probably want the height to be fixed. If you do something like `height: 300px; width: auto;`, that will retain the aspect ratio while maintaining a fixed height between images.

File: public/main.css [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+  color: rgb(32, 28, 28);
+  font-size: 15px;
+  line-height: 1.5;
+  margin-left: 80px;
+  margin-right: 80px;
+  text-align: center;
+}
+
+img {
```
Comment (jfarmer) [inline public/main.css:57]: You probably don't want to make _every_ `` tag on the site a block element. If you want to make every `` inside a carousel item, do something like...

```css
.carousel .item img {
  display: block;
}
```

**Review by jfarmer — COMMENTED** [review]
See if you can't fix the jittery carousel and misaligned navbar items in a Bootstrap-friendly way.

Don't try converting to 4.4, it'll take to much work. Continue with 3.4.
